### ISSUE TITLE
1 high severity issue when we install the plugin (CVE in json5)

### ISSUE THREAD

**Issue by messenjer** [import-js/eslint-plugin-import#2712]
Hello,

After installing the plugin, we have : 

```sh
1 high severity vulnerability
```

```sh
# npm audit report

json5  <1.0.2
Severity: high
Prototype Pollution in JSON5 via Parse Method - https://github.com/advisories/GHSA-9c47-m6qq-7p4h
fix available via `npm audit fix`
node_modules/tsconfig-paths/node_modules/json5

1 high severity vulnerability
```

The issue is https://github.com/advisories/GHSA-9c47-m6qq-7p4h in json5

It's fixed in `json5` https://github.com/json5/json5/commit/7774c1097993bc3ce9f0ac4b722a32bf7d6871c8 

This module is used by `tsconfig-path` and fixed in the latest version `4.1.2`  https://github.com/dividab/tsconfig-paths/blob/master/CHANGELOG.md

`Eslint-plugin-import` uses `tsconfig-path` version `3.14.1`, when I read the changelog there is no breaking change between the `3.14.1` and `4.1.2`. https://github.com/dividab/tsconfig-paths/blob/master/CHANGELOG.md 

May be we can update to the `tsconfig-path` version `4.1.2`. What do you think?

Thanks for the module.

Mathieu

**Comment by ljharb** [comment]
1. the vulnerability is in json5, not tsconfig-paths
1. we depend on tsconfig-paths v3, which depends on json5 v1
1. json5 v1 and v2 *both* have the fix published, so all you need to do is update your lockfile
1. this is not actually a valid vulnerability for this plugin anyways, so it's irrelevant
1. tsconfig-paths v4 is *absolutely* a breaking change because it drops, among others, node 4 - see #2447 

This is a duplicate of #2625; a duplicate of #2628; a duplicate of #2626; a duplicate of #2627; a duplicate of #2631; a duplicate of #2632; a duplicate of #2634; a duplicate of #2635; a duplicate of #2636; a duplicate of #2637; a duplicate of #2639; a duplicate of #2642; a duplicate of #2643; a duplicate of #2649; a duplicate of #2655; a duplicate of #2660; a duplicate of #2670; a duplicate of #2692; a duplicate of #2711.

All you need to do is install the plugin fresh, which should update your lockfile to use json5 v1.0.2.

**Comment by messenjer** [comment]
Apologies for the noise, thank you for the information.
